packages/storefront-sdk-plugins/commerce-queries/src/graphql/documents.ts

```diff
@@ -2804,6 +3009,184 @@ export const NavigationDocument = {
 		...NavigationItem_NavigationItemFragmentDoc.definitions
 	]
 } as unknown as DocumentNode<NavigationQuery, NavigationQueryVariables>;
+export const ProductCollectionEntriesDocument = {
+	kind: 'Document',
+	definitions: [
+		{
+			kind: 'OperationDefinition',
+			operation: 'query',
+			name: { kind: 'Name', value: 'productCollectionEntries' },
```

Comment:
Can you run `npm run codegen:all` so this updates this too.

Reply:
Yes sir! Thank you